File: modules/AcroStack.OpenIddictManagement/Dtos/OpenIddictTokenDto.cs
using System;

namespace AcroStack.OpenIddictManagement.Dtos;

/// <summary>
/// Token view for admin listing. Deliberately excludes the Payload column
/// (raw JWT/reference token material) so sensitive credentials never leave
/// the server.
/// </summary>
public class OpenIddictTokenDto
{
    public Guid Id { get; set; }

    public Guid? ApplicationId { get; set; }

    public Guid? AuthorizationId { get; set; }

    public string Subject { get; set; } = default!;

    public string Type { get; set; } = default!;

    public string Status { get; set; } = default!;

    public string? ReferenceId { get; set; }

    public DateTime? ExpirationDate { get; set; }
}

File: react/src/api/models/acroStack/openIddictManagement/dtos/OpenIddictTokenDto.ts
/* oxlint-disable */

/**
 * @description Token view for admin listing. Deliberately excludes the Payload column\r\n(raw JWT/reference token material) so sensitive credentials never leave\r\nthe server.
 * @type object
 */
export type AcroStackOpenIddictManagementDtosOpenIddictTokenDto = {
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  id?: string;
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  applicationId?: string | null;
  /**
   * @description
   * Format: `uuid`
   * @type string | undefined
   */
  authorizationId?: string | null;
  subject?: string | null;
  type?: string | null;
  status?: string | null;
  referenceId?: string | null;
  /**
   * @description
   * Format: `date-time`
   * @type string | undefined
   */
  expirationDate?: string | null;
};
